src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -74,14 +75,14 @@ public class DisputeController {
 	public ResponseEntity<List<DisputeListItem>> getAllDisputes(
 			@RequestParam(required = false)
 			@DateTimeFormat(pattern = "yyyy-MM-dd")
-			@Parameter(description = "If specified, will retrieve records older than this date (specified by yyyy-MM-dd)", example = "2022-03-15")
-			Date olderThan,
+			@Parameter(description = "If specified, will retrieve records newer than this date (specified by yyyy-MM-dd)", example = "2022-03-15")
+			Date newerThan,
 			@RequestParam(required = false)
 			@Parameter(description = "If specified, will retrieve records which do not have the specified status", example = "CANCELLED")
 			DisputeStatus excludeStatus) {
 		logger.debug("GET /disputes called");
 		logger.debug("Excluding status: {}", StructuredArguments.value("excludeStatus", excludeStatus));
-		List<DisputeListItem> disputeListItems = disputeService.getAllDisputes(olderThan, excludeStatus);
+		List<DisputeListItem> disputeListItems = disputeService.getAllDisputes(newerThan, excludeStatus);
```

Comment:
I think this is supposed to be **olderThan**. If the the database query is not doing this, then I think this is a bug.

The purpose of this variable to find all Disputes that have been unassigned for more than 1 hour and via an OpenShift cronjob to unassign them. ie. Find all Disputes where the assigned timestamp is older than 1 hour old.

See the [/disputes/unassign](https://github.com/bcgov/jag-traffic-courts-online/blob/731fbdae548e58a88be666539f4dbc7919a88d8a/src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java#L385) endpoint

Reply:
I think this `GET /disputes` endpoint's  filtering is different than `/disputes/unassign` endpoint. You're right about the unassign one and the parameter name should remain as `olderThan`. It was changed too because I did a find/replace all for the olderThan parameter name. I will change it back to `olderThan`. Good catch.

However, the getAllDisputes endpoint is irrelevant of unassign feature as far as I can see. Since there was no reported bug for that either, I think we should change the parameter name to match what database query is actually doing. It's weird that I don't even see that date parameter is used in Staff API to filter out returned disputes at all.